ndt7/results/file.go

```diff
@@ -105,27 +107,39 @@ type savedMeasurement struct {
 	// Measurement is the actual measurement to be saved.
 	Measurement model.Measurement `json:"measurement"`
 }
+*/
+
+// SaveClientMeasurement saves the |measurement| for archival data.
+func (fp *File) SaveClientMeasurement(measurement model.Measurement) {
+	fp.Data.ClientMeasurements = append(fp.Data.ClientMeasurements, measurement)
+}
+
+// SaveServerMeasurement saves the |measurement| for archival data.
+func (fp *File) SaveServerMeasurement(measurement model.Measurement) {
+	fp.Data.ServerMeasurements = append(fp.Data.ServerMeasurements, measurement)
+}
+
+// SaveMetadata writes |metadata| on the measurements file.
+func (fp *File) SaveMetadata(metadata metadata) {
+	fp.Data.ClientMetadata = metadata
+}
```

Comment:
(Maybe this is the non-native bias but) it confused me to see the `Save` verb used when data is not saved on disk but just added to a data structure. In fact, above, I find myself wondering what `SaveMetadata` did.

I'd suggest to either explain me why `save` is the right verb in this context or to use another verb. (I'd instinctively use `set` for setting a single variable and `append` for appending to a slice.)